packages/calypso-build/bin/build-package.js

```diff
@@ -8,22 +8,19 @@ const { execSync } = require( 'child_process' );
 
 const dir = process.cwd();
 const root = path.dirname( __dirname );
-const babelConfigFile = path.join( root, '..', '..', 'babel.config.js' );
 
 const inputDir = path.join( dir, 'src' );
 const outputDirEsm = path.join( dir, 'dist', 'esm' );
 const outputDirCommon = path.join( dir, 'dist', 'cjs' );
 
 console.log( 'Building %s', dir );
 
-// TODO: Drop `--config-file ${ babelConfigFile }` once we've moved `babel.config.js` to `packages/calypso-build`.
-execSync( `npx babel --config-file ${ babelConfigFile } -d ${ outputDirEsm } ${ inputDir }`, {
+execSync( `npx babel -d ${ outputDirEsm } ${ inputDir }`, {
 	env: Object.assign( {}, process.env, { CALYPSO_CLIENT: 'true' } ),
 	cwd: root,
 } );
 
-// TODO: Drop `--config-file ${ babelConfigFile }` once we've moved `babel.config.js` to `packages/calypso-build`.
-execSync( `npx babel --config-file ${ babelConfigFile } -d ${ outputDirCommon } ${ inputDir }`, {
+execSync( `npx babel -d ${ outputDirCommon } ${ inputDir }`, {
```

Comment:
Is it not necessary to specify the babel config in this package because it's resolved relative to where this is invoked?

Reply:
Apparently it is, after all :man_facepalming: This is what was causing the build failure.